public/js/main.js

```diff
@@ -52,11 +52,13 @@ $(function() {
 		});
 	});
 
-	$('input.datepicker').datepicker({
-		dateFormat: "dd-mm-yy"
+	flatpickr('input.datepicker', {
+		dateFormat: 'd-m-Y',
+		enableTime: false,
+		timeFormat: "H:i",
+		time_24hr: true,
```

Comment:
Can you remove the last comma? Older browser cant handle those (both arrays and objects).